### PR TITLE
Add elasticsearch tls support to spark

### PR DESCRIPTION
**Pull request by cgroschupp**

### CODE REVIEW

File: charts/jaeger/README.md [hunk 1]
```diff
@@ -223,137 +225,139 @@ helm install jaeger jaegertracing/jaeger \
...
-| `storage.elasticsearch.port` | Provisioned elasticsearch port| `9200` |
-| `storage.elasticsearch.scheme` | Provisioned elasticsearch scheme | `http` |
-| `storage.elasticsearch.usePassword` | Use password | `true` |
-| `storage.elasticsearch.user` | Provisioned elasticsearch user| `elastic` |
-| `storage.elasticsearch.nodesWanOnly` | Only access specified es host | `false` |
-| `storage.kafka.brokers` | Broker List for Kafka with port | `kafka:9092` |
-| `storage.kafka.topic` | Topic name for Kafka | `jaeger_v1_test` |
-| `storage.type` | Storage type (ES or Cassandra)| `cassandra` |
-| `tag` | Image tag/version | `1.16.0` |
+| Parameter                                                 | Description                                                                                                                                                      | Default                                 |
```
Comment (naseemkullah) [inline charts/jaeger/README.md]: Could we keep the spacing as it was before?
It is more readable in an IDE imho... BTW how did you space it all out?

File: charts/jaeger/README.md [hunk 2]
```diff
@@ -170,24 +170,26 @@ collector:
...
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: jaeger-tls
-data:
-  es.pem: |
-    -----BEGIN CERTIFICATE-----
-    <CERT>
-    -----END CERTIFICATE-----
+Generate configmap jaeger-tls:
```
Comment (naseemkullah) [inline charts/jaeger/README.md]: Is it not better to put this data in a secret rather than a cm?

Comment (cgroschupp) [reply charts/jaeger/README.md]: For my use case the configmap is ok, because it is only the certificate, not the private key.

Comment (cgroschupp) [reply charts/jaeger/README.md]: But with extraSecretMounts it is also possible to use a secret.

File: charts/jaeger/README.md (hunk 1 above)
Comment (cgroschupp) [reply charts/jaeger/README.md]: My IDE formats the markdown automatically.

Comment (naseemkullah) [reply charts/jaeger/README.md]: Oh I see, thanks.

Comment (cgroschupp) [reply charts/jaeger/README.md]: i have disabled the table autoformatting feature of my IDE and committed the changes

Comment (naseemkullah) [reply charts/jaeger/README.md]: Thanks! Hmmm maybe I should turn on mine. :)
 I will look into that but thanks for the info and reverting the change.

**Pull request merged**